### PR TITLE
i18n support

### PR DESCRIPTION
**Pull request by asdf404**
To translate the component, please pass `messages` property to component.
Where `messages` it's an object with following format:

```js
{
    children: 'Children',
    childrenAge: '{children, plural, =1 {Child age} other {Children ages}}',
    adults: 'Adults'
}

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,16 @@ Made for [Browserify.](http://browserify.org/)
...
+To translate the component, please pass `messages` property to component.
+Where `messages` it's an object with following format:
+
+```js
+{
+    children: 'Children',
+    childrenAge: '{children, plural, =1 {Child age} other {Children ages}}',
+    adults: 'Adults'
+}
+```
```
Comment (ikr) [inline README.md]: Please document the dependency on `Intl` and its poly-fill

File: package.json [hunk 2]
```diff
@@ -51,6 +51,8 @@
     "watchify": "^2.1.1"
   },
   "dependencies": {
-    "react": "^0.12.0"
+    "intl": "^0.1.4",
```
Comment (ikr) [inline package.json]: `intl` -> `devDependencies`

File: src/ChildrenInput.js [hunk 3]
```diff
@@ -4,18 +4,32 @@
...
 
     module.exports = React.createClass({
+        mixins: [IntlMixin],
         propTypes: {
             value: React.PropTypes.array,
             onChange: React.PropTypes.func
         },
 
         render: function () {
+            this.props.messages = this.props.messages ? this.props.messages : enMessages();
```
Comment (ikr) [inline src/ChildrenInput.js]: defaultProps please

File: src/OccupancyInput.js [hunk 4]
```diff
@@ -4,21 +4,38 @@
...
+        FormattedMessage = ReactIntl.FormattedMessage,
+        IntlMixin = ReactIntl.IntlMixin,
+
+        enMessages = function () {
+            return {
+                children: 'Children',
+                childrenAge: '{children, plural, =1 {Child age} other {Children ages}}',
+                adults: 'Adults'
+            };
+        },
```
Comment (ikr) [inline src/OccupancyInput.js]: Please extract enMessages into a node module, instead of copy-pasting it

File: src/OccupancyInput.js [hunk 5]
```diff
@@ -4,21 +4,38 @@
...
 
     module.exports = React.createClass({
+        mixins: [IntlMixin],
         propTypes: {
             value: React.PropTypes.object.isRequired,
             onChange: React.PropTypes.func.isRequired
         },
 
         render: function () {
+            this.props.messages = this.props.messages ? this.props.messages : enMessages();
```
Comment (ikr) [inline src/OccupancyInput.js]: defaultProps please

**Comment by asdf404** [comment]
@ikr fixed. Please reopen PR

**Pull request merged**